borg_import/rsnapshots.py

```diff
@@ -1,6 +1,20 @@
 import os
+import subprocess
 
 
 def get_snapshots(rsnapshot_root):
-    snapshots = os.listdir(rsnapshot_root)
-    return snapshots
+    output = []
+    for dirname, dirnames, filenames in os.walk(rsnapshot_root):
+        for subdir in dirnames:
+            output.append(os.path.join(dirname, subdir))
+        return output
+
+
+def get_disk_usage(rsnapshot_root):
+    p = subprocess.Popen(['du', '-h', '--max-depth=1', rsnapshot_root], stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                         universal_newlines=True)
+    output, error = p.communicate()
+    if error:
+        return error
+    else:
+        return output.splitlines()
```

Comment:
maybe either use some of the simpler calls from subprocess module or make your own small tool function that does what you usually want (also later for borg calls) and call it from here.